prod/westeurope/internal/api/functions_admin/storage/account/terragrunt.hcl

```diff
@@ -0,0 +1,22 @@
+# Internal
```

Comment:
Please instead of api/functions_admin/storage move all to api/storage_userbackups